src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -122,90 +122,122 @@
 			<!-- All open conversations list -->
 			<OpenConversationsList ref="openConversationsList" />
 
-			<!-- New Conversation dialog-->
+			<!-- New Conversation dialog -->
 			<NewConversationDialog ref="newConversationDialog" :can-moderate-sip-dial-out="canModerateSipDialOut" />
 
-			<!-- New Conversation dialog-->
+			<!-- New phone (SIP dial-out) dialog -->
 			<CallPhoneDialog ref="callPhoneDialog" />
+
+			<!-- New Pending Invitations dialog -->
+			<InvitationHandler v-if="isFederationEnabled" ref="invitationHandler" />
 		</div>
 
+		<NcAppNavigationItem v-if="pendingInvitationsCount"
+			class="invitation-button"
+			:name="t('spreed', 'Pending invitations')"
+			@click="showInvitationHandler">
+			<template #icon>
+				<AccountMultiplePlus :size="20" />
+			</template>
+			<template #counter>
+				<NcCounterBubble type="highlighted">
+					{{ pendingInvitationsCount }}
+				</NcCounterBubble>
+			</template>
+		</NcAppNavigationItem>
```

Comment:
maybe align with the conversation item margin or search box's ?

Reply:
It was aligned, might be the same issue from recent lib bump?  (cropped items)